activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -915,7 +914,7 @@ def configure_connection
         variables['wait_timeout'] = self.class.type_cast_config_to_integer(wait_timeout)
 
         # Make MySQL reject illegal values rather than truncating or blanking them, see
-        # http://dev.mysql.com/doc/refman/5.6/en/server-sql-mode.html#sqlmode_strict_all_tables
+        # http://dev.mysql.com/doc/refman/5.6/en/sql-mode.html#sqlmode_strict_all_tables
```

Comment:
@carlosantoniodasilva yes, previous link is unavailable or removed. I do it in separate PR as you suggest.